Admin/AdminHelper.php

```diff
@@ -37,25 +34,6 @@ public function __construct(Pool $pool)
     }
 
     /**
-     * @throws \RuntimeException
-     *
-     * @param \Symfony\Component\Form\FormBuilderInterface $formBuilder
-     * @param string                                       $elementId
-     *
-     * @return \Symfony\Component\Form\FormBuilderInterface
-     */
-    public function getChildFormBuilder(FormBuilderInterface $formBuilder, $elementId)
```

Comment:
Removing a public method is completely BC break.

BTW, why removing it?

If it's necessary, please make BC with deprecation system.